secure/handler/authorizationHandler.go

```diff
@@ -117,6 +124,7 @@ func (a AuthorizationHandler) Decorate(delegate http.Handler) http.Handler {
 			logging.MessageKey(), "request denied",
 			"validator-response", valid,
 			"validator-error", err,
+			"sat-client-id", satClientID,
```

Comment:
Don't forget that you can emit a `logger` with the `sat-client-id` prebaked:

`logger = log.With(logger, "sat-client-id", satClientID`

That way, any code that uses `logger` from there on out will get the `sat-client-id` field automatically.